### ISSUE TITLE
Responsive Navigation

### ISSUE THREAD

**Issue by andy5995** [Lyrics/lyrics.github.io#32]
What do you think about adding "reponsive navigation" using the [W3.CSS Framework?](https://www.w3schools.com/w3css/w3css_navigation.asp)

I added something like that to [one of my websites](https://remove-to-waste.info/) and should be able to take on this issue.

**Comment by snshn** [comment]
That navigation widget seems to look good, but until we have to integrate international alphabets it'll unlikely benefit the site, will only slow down the load and render time.

I built that web UI from scratch mostly for it to be extremely snappy and lightweight; it almost feels like it's a progressive web app — even though those are all separate static HTML files. GitHub Pages has a monthly bandwidth limit, so every little bit counts.

In this case less is more: if we try to improve it by using CSS frameworks and/or adding more DOM, it'll only become slower and harder to customize, resulting in being as sluggish and "sticky" as those other lyrics websites.

I'd say we could add more CSS to home and search pages, in separate from the rest of the site files, but those that belong to database structure and song pages have to really be as light as possible. Truth being told it's already not as fast as it was before I added dark CSS and some SVG assets; I've been meaning to give it a small refactor, but some features like that dark CSS just have to be there in 2020.

Also, this is not the only way to work with this database, there's [lyrics-api](https://github.com/Lyrics/lyrics-api) that makes it possible to self-host the database for music players, the site is mainly there for search engines and mobile use, it doesn't need any advanced features or enhancements really.
